sqlmesh/dbt/model.py

```diff
@@ -113,6 +115,13 @@ class ModelConfig(BaseModelConfig):
     def _validate_list(cls, v: t.Union[str, t.List[str]]) -> t.List[str]:
         return ensure_list(v)
 
+    @field_validator("check_cols", mode="before")
+    @classmethod
+    def _validate_check_cols(cls, v: t.Union[str, t.List[str]]) -> t.Union[str, t.List[str]]:
+        if isinstance(v, str) and v == "all":
```

Comment:
all is a special value here? what if it's upper case? what if you have a column called all?

Reply:
ah, this is dbt, i guess the last question doesn't hold then.

Reply:
This is dbt so we didn't create this contract. We use a `*` to help avoid that issue. 

Although from the docs it seems like it is expected to be lowercase I can just do a case-insensitive check here to be sure.